src/components/shared/card.tsx

```diff
@@ -172,58 +231,24 @@ const Card = ({ relatedItem, image }: Props): JSX.Element => {
         withDefault<ReactElement | null>(null)
     )
 
-    const parentIconStyles = (format: Format): SerializedStyles => css`
-        display:inline-block;
-        svg {
-            width: 0.875rem;
-            height: auto;
-            margin-left: auto;
-            margin-right: auto;
-            margin-top: 0.300rem;
-            display: block;
-        }
-    `;
-    const { inverted } = getPillarStyles(format.pillar);
-    const iconStyles = (format: Format): SerializedStyles => css`
-        width: 1.5rem;
-        height: 1.4375rem;
-        display: inline-block;
-        background-color: ${inverted};
-        border-radius: 50%;
-    `;
-
-    const icon = (itemType: RelatedItemType, format: Format): JSX.Element => {
-        if (itemType === RelatedItemType.GALLERY){
-            return <section css={parentIconStyles}>
-                    <span css={iconStyles}>< SvgCamera /></span>
-                   </section>;
-        } else {
-            return <section css={parentIconStyles} ></section>;
-        }
-    }
-
-    const metaDataStyles = (format: Format): SerializedStyles => css`
-        padding: 0 ${remSpace[2]};
-        min-height: 1.4375rem
-    `;
-
     const lastModified = relatedItem.lastModified?.iso8601;
     const date = lastModified ? relativeFirstPublished(fromNullable(new Date(lastModified))) : null;
 
     return <li css={[listStyles, cardStyles(relatedItem.type, format)]}>
-            <a css={anchorStyles} href={relatedItem.link}>
-                <section css={headingWrapperStyles}>
-                    <h3 css={headingStyles}>{relatedItem.title}</h3>
-                </section>
-                <section>
-                    <div css= {metaDataStyles}>
-                        {icon(relatedItem.type, format)} 
-                        {date}
-                    </div>
-                    <div css={imageWrapperStyles}>{img}</div>
-                </section>
-            </a>
-        </li>
+        <a css={anchorStyles} href={relatedItem.link}>
+            <section css={headingWrapperStyles}>
+                <h3 css={headingStyles}>{relatedItem.title}</h3>
+            </section>
+            <section>
+                <div css={metadataStyles}>
+                    {icon(relatedItem.type, format)}
```

Comment:
Since the icon parent doesn't appear to change, would it make sense to do:

```suggestion
                    <section css={parentIconStyles}>
                        {icon(relatedItem.type, format)}
                    </section>
```

and just have `icon` return the `span` (`null` in the default case)?

On a related note, what's the thinking behind this being a `section`? Not suggesting that it's wrong btw, just curious about the semantics you're conveying 🙂